components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go

```diff
@@ -62,6 +62,10 @@ func (p *ProvisionAzureEventHubStep) Run(operation internal.ProvisioningOperatio
 
 	hypType := hyperscaler.Azure
 
+	// TODO
+	operation.InputCreator.EnableComponent("KnativeEventingKafka")
+	//
+
```

Comment:
Is there still a todo or is it a leftover?
```suggestion
	operation.InputCreator.EnableComponent("KnativeEventingKafka")
```

Reply:
Done